TastiesApp/apps.py

```diff
@@ -1,6 +1,6 @@
 from django.apps import AppConfig
 
 
-class WebpageConfig(AppConfig):
+class TastiesappConfig(AppConfig):
```

Comment:
Please fix the name to 'Tasties**A**ppConfig' (Python convention).

Reply:
Thanks. Done.